docs/index.rst

```diff
@@ -6,20 +6,63 @@
 Welcome to the Documentation for CERN Analysis Preservation
 ============================================================
 
-These will be the CAP docs some time in the future.
+`CERN Analysis Preservation <https://analysispreservation.cern.ch>`_ is a data preservation service run at CERN currently targeted at storing analysis information from the four largest LHC experiments.
+It is an open source project licensed under the `GNU Public License 2.0 <https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html>`_, yet the information stored will be private.
 
-Table of Content
-----------------
+We want to make it easier for physicists who are meant to have access to this analysis information to find it, understand it and use it just as they have done before but with less effort involved. Plus, we want this to be possible as you are doing the analysis, during approval, after publishing the paper and even ten years afterwards - just in case something might come up that requires expertise for your analysis. In short:
```

Comment:
"and even ten years afterwards" - might it be better to describe it more generally without raising or lowering expectations ("**even 10 years**")? Smth like "and even in many years in the future when technologies have changed"